src/views/helpers/HomepageHelper.elm

```diff
@@ -0,0 +1,14 @@
+module HomepageHelper exposing (..)
```

Comment:
I would probably opt to include this the `Homepage` module until it made sense to break it out into a new module that makes sense.